### ISSUE TITLE
-n option doesn't work when combined with --extend-rows

### ISSUE THREAD

**Issue by atsalolikhin-spokeo** [alexhallam/tv#140]
Example:

```
[~] [py-3.9.7] $ tv -n 2 /tmp/example.csv

      tv dim: 26 x 1
      one
1     two
2     three
      … with 24 more rows
[~] [py-3.9.7] $ tv -n 2 --extend-rows /tmp/example.csv

      tv dim: 26 x 1
      one
1     two
2     three
3     four
4     five
5     one
6     two
7     three
8     four
9     five
10    one
11    two
12    three
13    four
14    five
15    one
16    two
17    three
18    four
19    five
20    one
21    two
22    three
23    four
24    five
25    six
26    last line

[~] [py-3.9.7] $
```

**Comment by alexhallam** [comment]
Thanks. I was not aware of this issue.

**Comment by alexhallam** [comment]
Here is a fix on #142

![image](https://user-images.githubusercontent.com/9298693/180616089-d9567cde-50ec-44e2-ad44-22c2a935fa37.png)
